app/templates/pdf/order_invoice.html

```diff
@@ -1,126 +1,228 @@
-    <br>
-    <table class="table with-datatable" id="orders-table" width="100%">
-        <thead>
-        <tr>
-            <th>
-                {{ ("Order") }}
-            </th>
-            <th>
-                {{ ("Order Details") }}
-            </th>
-            <th>
-                {{ ("Billing Info") }}
-            </th>
-        </tr>
-        </thead>
-        <tbody>
-        {% if order.status != "deleted" %}
-            <tr class="padded">
-                <td><br>
-                        Order Number    :<br>
-                        Order By        :<br>
-                        Order Status    :<br>
-                        Quantity        :<br>
-                        Total Amount    :<br>
-                        Payment Mode    :<br>
-                        Discount Code   :<br>
-                </td>
-                <td><br>
-                        <strong>{{ order.get_invoice_number() }}</strong><br>
-                        {% if order.user and order.user.first_name and order.user.last_name %}
-                            {{ order.user.fullname }}
-                        {% elif order.user %}
-                            {{ order.user.email }}
-                        {% else %}
-                            {{ ('Information unavailable') }}
-                        {% endif %}<br>
-                        {% if order.status == 'completed' %}
-                            {{ order.status | capitalize }}
-                        {% elif order.status == 'pending' or order.status == 'initialized' %}
-                            {{ ("Pending") }}
-                        {% elif order.status == 'placed' %}
-                            {{ order.status | capitalize }}
-                        {% elif order.status == 'cancelled' %}
-                            {{ order.status | capitalize }}
-                        {% else %}
-                            {{ order.status | capitalize }}
-                        {% endif %}<br>
-                        {{ order.tickets_count }}<br>
-                        {{ event.payment_currency | currency_symbol }}{{ order.amount | money }}<br>
-                        {% if order.status == 'completed' %}
-                            {{ order.paid_via | capitalize }}
+            <hr noshade>
+
+            <table class="table">
+                    <thead class="black white-text">
+                      <tr>
+                        <th scope="col">Name</th>
+                        <th scope="col">Price</th>
+                        <th scope="col">Quantity</th>
+                        <th scope="col">Sub-Total(net)</th>
+                        <th scope="col">VAT</th>
+                        <th scope="col">Tax Amount</th>
+                      </tr>
+                    </thead>
+                    <tbody>
+                    {% for ticket in tickets %}
+                      <tr>
+                        <td>{{ ticket.name }}</td>
```

Comment:
Use custom alignment.
`<td style="text-align:center">` This style should match the `<thead>` tag's alignment which you are currently using

Reply:
@uds5501 TYup.